blueprints/ember-segment/index.js

```diff
@@ -1,37 +0,0 @@
-'use strict';
-
-module.exports = {
-  normalizeEntityName: function() { },
-
-  beforeInstall: function() {
-    var eventName = 'gatherPageAnalytics';
-
-    var reopens = [
```

Comment:
I don't understand why you deleted this and then suggest adding the same code to `router.js` in `README.md`